Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -631,8 +631,22 @@ extension BrowserViewController: UIScrollViewDelegate {
 
 extension BrowserViewController: TabManagerDelegate {
     func tabManager(tabManager: TabManager, didSelectedTabChange selected: Browser?, previous: Browser?) {
+        // Remove the old accessibilityLabel. Since this webview shouldn't be visible, it doesn't need it
+        // and having multiple views with the same label confuses tests.
+        if let wv = previous?.webView {
+            wv.accessibilityLabel = ""
+        }
```

Comment:
Hm, this feels a bit hacky, but maybe this is the right way to do this. Nit: `nil` instead of `""`?